### PR TITLE
dt: parse input with numbers and letters properly

### PR DESCRIPTION
**Pull request by jumbowhales**
still doesn't parse input with commas properly

### CODE REVIEW

File: chat-plugins/info.js [hunk 1]
```diff
@@ -362,7 +362,7 @@ exports.commands = {
 		let targetId = toId(target);
 		if (!targetId) return this.parse('/help data');
 		let targetNum = parseInt(targetId);
-		if (!isNaN(targetNum)) {
+		if (!isNaN(targetNum) && targetNum == targetId) {
```
Comment (Zarel) [inline chat-plugins/info.js]: `'' + targetNum === targetId` might be what you're looking for.

Comment (jumbowhales) [inline chat-plugins/info.js]: is ``'' + targetNum`` preferable over ``targetNum.toString``?

**Comment by jumbowhales** [comment]
this is why you shouldn't edit while retarded, sorry

File: chat-plugins/info.js (hunk 1 above)
Comment (Zarel) [inline chat-plugins/info.js]: very strongly

**Pull request closed without merging**